tests/test-rmd.R

```diff
@@ -9,10 +10,21 @@ if (Sys.getenv('NOT_CRAN') == 'true') local({
   for (f in list.files('rmd', '[.]Rmd$', full.names = TRUE)) {
     rmarkdown::render(f, envir = globalenv(), quiet = TRUE)
   }
+ 
+  html_issues = simplify_html_validation(
+    validate_html(list.files("rmd", ".html$", full.names = TRUE))
+  )
+  if(nrow(html_issues) > 0)
+    stop("HTML issues detected in ", paste(html_issues$file, collapse = ', '))
```

Comment:
Can we also show `html_issues$messages` here?

Reply:
Done. I also pushed to the breakage demo branch. You can see the result here: https://github.com/katrinabrock/bookdown/actions/runs/14272976278/job/40010183970?pr=1#step:13:252